java/arcs/jvm/host/JvmSchedulerProvider.kt

```diff
@@ -61,7 +61,7 @@ class JvmSchedulerProvider(
                     val thread = threads[threadIndex % maxThreadCount]
                     if (thread != null && thread.isAlive) return@newSingleThreadExecutor thread
 
-                    if (thread?.isAlive == false) log.warning {
+                    if (thread?.isAlive == false) log.info {
                         "Creating a new thread (index: ${threadIndex % maxThreadCount}) because " +
```

Comment:
Is this because warnings are preserved more than info is?

Reply:
I think it's more about the ramifications intended for a reader to have in mind when looking at logcat through the lens of Android. I think we might want to map things semantically more medium term, IMO one could argue this really is warning, but this is at least a short term step. We should discuss further, see also the bug in description for more background.